src/modules/blocks/rsvp/template.js

```diff
@@ -22,6 +24,34 @@ import MoveModal from '@moderntribe/tickets/elements/move-modal';
 import { Card } from '@moderntribe/tickets/elements';
 import './style.pcss';
 
+/**
+ * Get the block controls for the RSVP block.
+ *
+ * @since TBD
+ * @return {Array} The block controls.
+ */
+function getRSVPBlockControls() {
+	const controls = [];
+
+	/**
+	 * Filters the RSVP block controls.
+	 *
+	 * @since TBD
+	 * @param {Array} controls The existing controls.
+	 */
+	return applyFilters( 'tec.tickets.blocks.RSVP.Controls', controls );
+}
+
```

Comment:
I think we need doc blocks for `RSVPControls` and `RSVP` since they are components.